pkg/apis/v1beta1/nodepool_validation.go

```diff
@@ -30,7 +30,7 @@ func (in *NodePool) SupportedVerbs() []admissionregistrationv1.OperationType {
 	}
 }
 
-func (in *NodePool) Validate(_ context.Context) (errs *apis.FieldError) {
+func (in *NodePool) WebhookValidate(_ context.Context) (errs *apis.FieldError) {
```

Comment:
Oh wait, I'm realizing that you can't do this now. You need to do `Validate` to conform to the webhook interface. You may have to call the other function `RuntimeValidate()` or similar for now until we drop the webhook support